### PR TITLE
Adding spellcasting for Ice Mpehit

### LINKED ISSUE #222: Ice Mephit - Potential Issue with Innate Spellcasting.
So there's a VERY significant difference between the Ice Mephit and Drow with regards to the "Innate Spellcasting" ability.

In the Drow's JSON data, Innate Spellcasting turns into a full spell block with a "spellcasting" block listing DC, Affecting Ability Score and a full "spells" block.

Conversely the Ice Mephit's "Innate Spellcasting" block is exceptionally limited.  There is no "spells" block, the only identifier for what spell the Mephit can cast is in the description of the Innate Spellcasting block itself (so no way to programmatically pull that out) and the whole block is missing everything the drow block has.  Usage is up 2 levels in the chain from the Drow's side:

Drow:  Innate Spellcasting -> spellcasting -> spells -> usage
Ice Mephit:  Innate Spellcasting -> usage

Is this intended?

### PR DESCRIPTION
**Pull request by bagelbits**
## What does this do?
Adds spellcasting info to Ice Mephit.

## How was it tested?
CI

## Is there a Github issue this is resolving?
This resolves #222 

## Here's a fun image for your troubles
image

### CODE REVIEW

File: src/5e-SRD-Monsters.json [hunk 1]
```diff
@@ -20436,9 +20436,24 @@
...
-          "type": "per day",
-          "times": 1
+        "spellcasting": {
+          "ability": {
+            "name": "CHA",
+            "url": "/api/ability-scores/cha"
+          },
+          "components_required": [],
+          "spells": [
+            {
```
Comment (caldane) [inline src/5e-SRD-Monsters.json]: mis-aligned open curly brace

File: src/5e-SRD-Monsters.json [hunk 2]
```diff
@@ -20436,9 +20436,24 @@
...
+              "name": "Fog Cloud",
+              "url": "/api/spells/fog-cloud",
+              "usage": {
+                "usage": {
```
Comment (caldane) [inline src/5e-SRD-Monsters.json]: unnecessary nesting

Comment (bagelbits) [reply src/5e-SRD-Monsters.json]: Fixed!

File: src/5e-SRD-Monsters.json (hunk 1 above)
Comment (bagelbits) [reply src/5e-SRD-Monsters.json]: It's.... not?

Comment (bagelbits) [reply src/5e-SRD-Monsters.json]: Or maybe it was but I fixed it.

**Review by caldane — APPROVED** [review]

**Pull request merged**